cmd/module_test.go

```diff
@@ -102,11 +102,11 @@ builders:
 			Ω(getTestPath("result", ".mta_mta_build_tmp", "node-js", "data.zip")).Should(BeAnExistingFile())
 		})
 
-		It("stand along build Command", func() {
+		It("stand alone build Command", func() {
 			soloBuildModuleCmdModule = "node-js"
 			soloBuildModuleCmdSrc = getTestPath("mta")
 			soloBuildModuleCmdTrg = getTestPath("result")
-			Ω(soloBuildModuleCmd.RunE(nil, []string{})).Should(Succeed())
+			soloBuildModuleCmd.Run(nil, []string{})
```

Comment:
If you change the command to return an error again please change this back

Reply:
i don't return